cmd/run.go

```diff
@@ -268,7 +268,13 @@ a commandline interface for interacting with it.`,
 				}
 				precision := 100 * time.Millisecond
 				atT := engine.Executor.GetTime()
-				if endT := engine.Executor.GetEndTime(); endT.Valid {
+
+				endT := engine.Executor.GetEndTime()
+				if stages := engine.Executor.GetStages(); len(stages) > 0 {
```

Comment:
Right, I forgot `SumStages()` returns a `NullDuration`, just make this an `endT := SumStages(...); endT.Valid`.